src/components/BooksList.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+function BooksList({ bookData }) {
+  const books = bookData.map(book => (
+    <tr key={book.id}>
+      <Book id={book.id} title={book.title} category={book.category} />
+    </tr>
+  ));
+
+  return (
+    <div>
+      <table className="table-auto">
+        <thead>
+          <tr>
+            <th>Book ID</th>
+            <th>Title</th>
+            <th>Category</th>
+          </tr>
+        </thead>
+        <tbody>
+          {books}
```

Comment:
**REQUIRED**
* It is placed in `src/containers` directory. Let's separate presentational and container elements.